manifests/init.pp

```diff
@@ -21,6 +21,9 @@
 # @param policy_default When managing plugin policies this will be the default allow/deny
 # @param site_policies Policies to apply to all agents after any module specific policies
 # @param rpcutil_policies Policies to apply to the special rpcutil agent
+# @param manage_package Install mcollective package on this node
+# @param package_name The name of the package to install
```

Comment:
maybe add here `The name of the package to install if manage_package is enabled`